### PR TITLE
Fixed a few issues that cropped up from testing (Rampfitoutput & WFI_ing_photom).

### PR DESCRIPTION
**Pull request by PaulHuwe**
Ticket to fix a couple inaccuracies and omissions in rampfitoutput and wfi_img_photom.

### CODE REVIEW

**Review by ddavis-stsci — APPROVED** [review]
LGTM

File: src/rad/resources/schemas/reference_files/wfi_img_photom-1.0.0.yaml [hunk 1]
```diff
@@ -10,9 +10,13 @@ properties:
...
     allOf:
       - $ref: ref_common-1.0.0
+      - type: object
+        properties:
+          reftype:
+            type: string
+            enum: [PHOTOM]
   phot_table:
     title: Photometric flux conversion factors table
-    type: object
```
Comment (eslavich) [inline src/rad/resources/schemas/reference_files/wfi_img_photom-1.0.0.yaml:15]: Why is the type being removed?  I don't think it's valid to have `patternProperties` without `type: object`

File: src/rad/resources/schemas/rampfitoutput-1.0.0.yaml [hunk 2]
```diff
@@ -34,7 +34,7 @@ allOf:
     pedestal:
```
Comment (eslavich) [inline src/rad/resources/schemas/rampfitoutput-1.0.0.yaml:34]: Should we change this schema URI and filename to `ramp_fit_output-1.0.0` so we end up with better consistency in capitalized words in roman_datamodels?

Comment (PaulHuwe) [reply src/rad/resources/schemas/rampfitoutput-1.0.0.yaml:34]: I can't think of a good not to. I will change the file_name.

File: src/rad/resources/schemas/reference_files/wfi_img_photom-1.0.0.yaml (hunk 1 above)
Comment (PaulHuwe) [reply src/rad/resources/schemas/reference_files/wfi_img_photom-1.0.0.yaml:15]: Because python throws errors about keys of type object.

**Review by eslavich — APPROVED** [review]
LGTM

**Pull request merged**